lib/cli/generators/POLYMER/index.js

```diff
@@ -14,20 +20,27 @@ export default async npmOptions => {
 
   packageJson.dependencies = packageJson.dependencies || {};
   packageJson.devDependencies = packageJson.devDependencies || {};
-  packageJson.devDependencies['@storybook/polymer'] = storybookVersion;
 
-  await installBabel(npmOptions, packageJson);
+  packageJson.scripts = packageJson.scripts || {};
+  packageJson.scripts.storybook = 'start-storybook -p 6006';
+  packageJson.scripts['build-storybook'] = 'build-storybook';
+
+  writePackageJson(packageJson);
 
+  let webpackDep = [];
   if (
     !packageJson.dependencies['polymer-webpack-loader'] &&
     !packageJson.devDependencies['polymer-webpack-loader']
   ) {
-    packageJson.devDependencies['polymer-webpack-loader'] = polymerLoaderVarion;
+    webpackDep = [`polymer-webpack-loader@${polymerLoaderVarion}`];
```

Comment:
Why have a distinction on what this devDependency is for?

Reply:
I think having a single list to then later install would make more sense?

Reply:
oh this one fell out of the normal set of storybook packages we were adding in other generators and was unique to this polymer one, so that's just how my brain was referencing it. But yea, no real need for a distinction here 👍